WearOAuth/oauth-device-grant/build.gradle

```diff
@@ -14,49 +14,67 @@
  * limitations under the License.
  */
 plugins {
-    id 'com.android.application'
-    id 'org.jetbrains.kotlin.android'
+  id 'com.android.application'
+  id 'org.jetbrains.kotlin.android'
 }
 
 android {
-    compileSdk 34
+  compileSdk 34
+  namespace "com.example.android.wearable.oauth.devicegrant"
 
-    namespace "com.example.android.wearable.oauth.devicegrant"
+  defaultConfig {
+    applicationId "com.example.android.wearable.oauth.devicegrant"
+    minSdk 26
+    targetSdk 33
+    versionCode 1
+    versionName "1.0"
+  }
 
-    defaultConfig {
-        applicationId "com.example.android.wearable.oauth.devicegrant"
-        minSdk 26
-        targetSdk 33
-        versionCode 1
-        versionName "1.0"
+  // Used to make the direct HTTP call for the token exchange. This will eventually be removed
+  // when the sample is updated to use server-side redirects.
+  useLibrary "org.apache.http.legacy"
+
+  buildTypes {
+    release {
+      minifyEnabled false
+      proguardFiles getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro"
     }
+  }
 
-    // Used to make the direct HTTP call for the token exchange. This will eventually be removed
-    // when the sample is updated to use server-side redirects.
-    useLibrary "org.apache.http.legacy"
+  compileOptions {
+    sourceCompatibility JavaVersion.VERSION_17
+    targetCompatibility JavaVersion.VERSION_17
+  }
 
-    buildTypes {
-        release {
-            minifyEnabled false
-            proguardFiles getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro"
-        }
-    }
-    compileOptions {
-        sourceCompatibility JavaVersion.VERSION_17
-        targetCompatibility JavaVersion.VERSION_17
-    }
-    kotlinOptions {
-        jvmTarget = JavaVersion.VERSION_17.majorVersion
-    }
+  kotlinOptions { jvmTarget = JavaVersion.VERSION_17.majorVersion }
+  buildFeatures {compose true }
+  composeOptions {kotlinCompilerExtensionVersion compose_version }
```

Comment:
nit: `{isFormattingSquished true }`

Reply:
lol

Reply:
Is the best practice to keep these multi-line?  Or is it just that I omitted white space?

Reply:
multiline-Ihaven'tseenthislayoutbefore }

there are 0-n settings, and this only works for the special case of 1

Reply:
Gotcha!  Done.